.circleci/config.yml

```diff
@@ -1,20 +1,79 @@
-# CircleCI 2.0 configuration file
+# CircleCI 2.1 configuration file
 #
 #
-version: 2
-jobs:
-  build:
+version: 2.1
+executors:
+  womtool-executor:
     docker:
       # specify the version you desire here
       - image: broadinstitute/womtool:47-b36d920
         entrypoint: /bin/bash
     working_directory: ~/palantir-workflows
 
-    environment:
-      # Customize the JVM maximum heap limit
+commands:
+  get-cromwell-jar:
+    description: "Download cromwell jar"
+    steps:
+      - run: wget https://github.com/broadinstitute/cromwell/releases/download/47/cromwell-47.jar
```

Comment:
you could also run the cromwell jobs in a different "job" that has the other docker (the cromwell one)

Reply:
Since running cromwell results in calling docker, circleci requires treating differently if done in a docker (ie, https://circleci.com/docs/2.0/building-docker-images/).  Among other things, this requires installing docker manually.  All in all, doesn't seem worth it to me.